types/js-cookie/index.d.ts

```diff
@@ -79,7 +79,7 @@ declare namespace Cookies {
          * or SDK. Note: The noConflict method is not necessary when using
          * AMD or CommonJS, thus it is not exposed in those environments.
          */
-        noConflict(): CookiesStatic;
+        noConflict?(): CookiesStatic;
```

Comment:
Is this method really optional?

Reply:
As stated in the line above: 
`The noConflict method is not necessary when using AMD or CommonJS, thus it is not exposed in those environments.`
The `noConflict` method only exists in a browser environment and is thus optional.

Reply:
Fair enough. Missed this one.